packages/EasyEntityChange/src/DataTransferObjects/ChangedEntity.php

```diff
@@ -0,0 +1,59 @@
+<?php
+declare(strict_types=1);
+
+namespace EonX\EasyEntityChange\DataTransferObjects;
+
+/**
+ * This DTO represents a single entity which has changed. Subclasses define the operation
+ * that occurred on the entity.
+ */
+abstract class ChangedEntity
+{
+    /**
+     * @phpstan-var class-string
```

Comment:
Why do we need PHPStan annotations? Especially on things that are already defined by their PHPDoc or type hinted by the code.

Reply:
Because class-string is not string - it explictly limits the string inputs to be instances of classes (which is what we want here)

Reply:
I would just annotate the @var as `class-string` but it will break phpstorm. The solution the phpstan/psalm communities have come up with is a prefixed annotation until IDEs support more advanced types